src/index.html

```diff
@@ -15,13 +15,404 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="styles/index.scss"
```

Comment:
The `href` attribute points to a SCSS file. Ensure that this file is compiled to CSS and linked correctly, as browsers cannot directly interpret SCSS.